src/ExtendedXmlSerializer/ExtendedXmlSerializerConfig.cs

```diff
@@ -132,6 +134,11 @@ void IExtendedXmlSerializerConfig.WriteObject(XmlWriter writer, object obj)
             _serializer(writer, (T) obj);
         }
 
+        public bool IsSatisfiedBy( Type type )
+        {
+            return specification( type );
```

Comment:
Why you created specification field instead of just write:
```C#
        public bool IsSatisfiedBy( Type type )
        {
            return Type.GetTypeInfo().IsAssignableFrom(type);
        }
```
I think it probably would be simpler? What do You think about it?